move-fast-and-break-things/aibyss
# Fix: #37 Fix the typo of bug

## How does this PR do?


- Fixes the module import error by correcting the path for getGameStats in api/state.get.ts
- Adds a mock getGameStats function in database/index.ts to simulate game data retrieval.
- Refactors the API handler to compute game ratings based on fetched statistics.
- Provides a basic TypeScript configuration (tsconfig.json) to handle module resolution and type checking (if applicable).


Fixes #37 

## What type of changes?

- Bug Fix: Corrected the module import path in api/handler.ts.
- New Feature: Added mock getGameStats function in database/index.ts to simulate game data.
- Refactor: Improved the API handler logic for fetching stats and computing ratings.

##How should be this tested

- Run the API and verify game stats are fetched correctly.
- Confirm accurate rating calculations from the fetched stats.
- Check for import errors in api/state.get.ts.
